app/models/user.rb

```diff
@@ -224,6 +225,16 @@ def cached_following_podcasts_ids
     end
   end
 
+  # TODO: (Alex Smith) - Adjust TTL and limit query size post Codeland
+  def cached_subscription_source_article_ids
+    cache_key = "user-#{id}-#{subscribed_to_user_subscriptions_count}/subscription_source_article_ids"
+    Rails.cache.fetch(cache_key, expires_in: 12.hours) do
```

Comment:
Very open to changing the time here. Just stuck with the `12.hours` trend. It'll be ignored anyway with `subscribed_to_user_subscriptions_count` changing.